packages/subiquity_client/lib/subiquity_client.dart

```diff
@@ -231,8 +231,19 @@ class SubiquityClient {
     await checkStatus("setStorage(${jsonEncode(config)})", response);
   }
 
-  Future<void> reboot() async {
-    final request = Request('POST', Uri.http('localhost', 'reboot'));
+  Future<void> reboot({bool immediate = false}) async {
+    final request = Request(
+        'POST',
+        Uri.http('localhost', 'shutdown',
+            {'mode': '"REBOOT"', 'immediate': '$immediate'}));
+    await _client.write(request);
+  }
+
+  Future<void> shutdown({bool immediate = false}) async {
+    final request = Request(
+        'POST',
+        Uri.http('localhost', 'shutdown',
+            {'mode': '"POWER_OFF"', 'immediate': '$immediate'}));
```

Comment:
The final mode term got changed to "POWEROFF" instead of having the underscore.

Reply:
Thanks